### PR TITLE
fix taosBenchmark document error

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

File: taosbenchmark-user-manual.md [hunk 1]
```diff
@@ -114,74 +110,79 @@ taosBenchmark can test TDengine's insertion, query and subscription
...
+||[interlace_rows](#interlace-rows) | The number of interleaved rows per child table, default is 0.
+|| [num_of_records_per_req](#record-per-request) | The number of records per request, the default is 30000.
+|| [prepare_rand](#prepared-rand) | The number of randomly generated data, the default is 10000 |
+|| | chinese | Both nchar and binary are rand Chinese, default is no.
+| dbinfo | name | The database name, required
+| dbinfo | drop | Whether to delete the database before inserting tests.
+| dbinfo | replica | The number of replicas, the default value is 1.
+| dbinfo | days | The span of time to store data in the file, the default value is 10.
+| dbinfo | cache | The size of the memory blocks in MB, the default value is 16.
+| dbinfo | blocks | The number of memory blocks per vnode(ttsdb) for the cache size, the default is 6.
```
Comment (sangshuduo) [inline taosbenchmark-user-manual.md:142]: is ttsdb correct?

File: taosbenchmark-user-manual.md [hunk 2]
```diff
@@ -307,58 +308,161 @@ taosBenchmark can test TDengine's insertion, query and subscription
...
+  Only available when [insert_mode](#insert-mode) is taosc, stmt, rest mode, no super table is created, only normal table is created.
+
+
+- #### childtable limit
+
+  Only works if childtable_exists is yes, for use with the statement
+  ``` select tbname from stable limit x offset y;```
+The value x of limit in the statement ``` is the number of existing child tables to get inserted.
+
+- #### childtable offet
```
Comment (sangshuduo) [inline taosbenchmark-user-manual.md:425]: offset

File: taosbenchmark-user-manual.md [hunk 3]
```diff
@@ -307,58 +308,161 @@ taosBenchmark can test TDengine's insertion, query and subscription
...
 
--   #### -b/--data-type
 
-  same as -A/--tag-type, but for columns, and default is FLOAT,INT,FLOAT
+- #### type
 
--   #### -O/--disorder
+  Please refer to the officially supported data types (https://www.taosdata.com/docs/cn/v2.0/taos-sql#data-type) for optional values.
+  
+  Note: JSON data type is special, only in tags can be selected, and there is only one JSON tag column, at this time [count](#count) and [len](#len) represent the meaning of the number of key-value pairs within the JSON tag and the length of the value of each kv pair, respectively. The default value is string.
```
Comment (sangshuduo) [inline taosbenchmark-user-manual.md:452]: KV

File: taosbenchmark-user-manual-CN.md [hunk 4]
```diff
@@ -299,56 +308,159 @@ taosBenchmark 是一个用于 TDengine 的性能测试的应用程序。taosBenc
...
+## 参数具体说明
+#### [返回](#taosbenchmark)
 
-  只有当隔行扫描(-B/-interlace-rows)大于0时才起作用。该
-  意味着线程在为每个子表插入隔行扫描记录后，会在该时间段内睡觉。
-  隔行扫描的时间。
+- #### insert mode
 
-- #### -B/--interlace-rows
+  可选项有taosc, rest, stmt, sml, sml-rest, 分别对应c接口，restful，参数绑定，schemaless的c接口，taosadaptor schemaless写入
```
Comment (sangshuduo) [inline taosbenchmark-user-manual-CN.md:316]: taosAdapter

File: taosbenchmark-user-manual-CN.md [hunk 5]
```diff
@@ -299,56 +308,159 @@ taosBenchmark 是一个用于 TDengine 的性能测试的应用程序。taosBenc
...
+  仅当 [insert_mode](#insert-mode) 为 taosc, stmt, rest 模式下可以使用，不创建超级表，只创建普通表。
+
+
+- #### childtable limit
+
+  仅当 childtable_exists 为 yes 时生效，为使用语句
+  ``` select tbname from stable limit x offset y;```
+中的limit的值 x，即为获取插入的现有的子表数量。
+
+- #### childtable offet
```
Comment (sangshuduo) [inline taosbenchmark-user-manual-CN.md:425]: offset

File: taosbenchmark-user-manual-CN.md [hunk 6]
```diff
@@ -299,56 +308,159 @@ taosBenchmark 是一个用于 TDengine 的性能测试的应用程序。taosBenc
...
+
+  仅当 childtable_exists 为 yes 时生效，为使用语句
+  ``` select tbname from stable limit x offset y;```
+中的 offset的值y，即为获取插入子表的偏移量。
+
+- #### sample file
+
+  是否使用以 csv 格式的数据作为数据源，仅当data_source 为 sample 时生效。
+  
+  注：这里与最终生成的数据不同，最终数据源的数据与[prepared_rand](#prepared-rand)的值有关，若 csv 文件内的数据行数小于prepraed_rand，那么会循环读取 csv 文件数据直到与prepared_rand相同，若大于，则会只读取 prepared_rand 个数的行的数据。
```
Comment (sangshuduo) [inline taosbenchmark-user-manual-CN.md:435]: prepared

**Review by sangshuduo — APPROVED** [review]

**Review by afwerar — APPROVED** [review]
LGTM

**Review by sangshuduo — CHANGES_REQUESTED** [review]
see comments

**Comment by zhaoyanggh** [comment]
> see comments

fixed

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**